pages/common/ksh.md

```diff
@@ -1,21 +1,25 @@
 # ksh
 
 > Korn Shell, a Bash-compatible command-line interpreter.
-> See also `histexpand` for history expansion.
+> See also: `histexpand` (history expansion).
```

Comment:
I don't really like this change. What's wrong with the previous sentence?

Reply:
I think we don't need to mention `history expansion` at all or we can keep it as the original version itself.